types/react-google-recaptcha/react-google-recaptcha-tests.tsx

```diff
@@ -10,11 +10,14 @@ const invisibleRecaptcha: React.SFC = () => {
             ref={recaptchaRef}
             sitekey="xxx"
             size="invisible"
+            asyncScriptOnLoad={() => {
+                console.log('test');
```

Comment:
```suggestion
```
can you please remove this, apply prettier formatting on the file and remove `DOM` from `tsconfig.json`?
There is nothing in DT package that reuqires DOM (unless `console.log` is being used in tests).
This will improve CI tests for package and interop with other packages (less dependencies)

Reply:
Will do, but how should I test the function then?

Reply:
It's tested durign CI by virtual compilation and TS linter (there is no actual output and there is no console object really). `console` is not under the test. If your test compiles fine with this property, that's it for added property. You can use non-JSX syntax to verify componnent has this property accessible, but JSX syntax fits the bill here. If a callback requies params, you would test params within a block to be a specific derived type, here, this one is parameterless, you've just tested property accepting empty callback. You can move that callback defnition up to be assigned to `const` somwhere with a type declaration, but this one is ok.